clj-refactor.el

```diff
@@ -149,7 +152,10 @@ with the middleware."
 
 (defvar cljr--nrepl-ops
   '("artifact-list" "artifact-versions" "clean-ns" "configure" "find-symbol"
-    "find-unbound" "hotload-dependency" "resolve-missing"))
+    "find-unbound" "hotload-dependency" "resolve-missing" "version"))
+
+(defvar cljr-version "1.0.5-SNAPSHOT"
```

Comment:
haha, I'm the one being slow here.  I was so annoyed by this duplication, with the version information appearing both in a var and in the header.  I spent like 10 mins trying to find a way to use `eval-when-compile` to extract the info from the header and put it into the var.

After all that I didn't see the literal duplication when you pointed it out.  Fixed in https://github.com/clojure-emacs/clj-refactor.el/commit/b0f1abaae0ea80b4ebeb45ac2f3474e426e050de